docs/development/documentation_guidelines.rst

```diff
@@ -6,15 +6,43 @@ Documentation Guidelines
 
 High-quality and consistent documentation is very important at TARDIS. It allows new users to find out how to do something specific using TARDIS, as well as helps developers (like you!) to understand the best practices.
 
-TARDIS uses the popular Python documentation generator `Sphinx <https://www.sphinx-doc.org/>`_. Sphinx translates a set of plain text source files (often written in `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_) to HTML files, automatically producing cross-references, indices, etc. If you haven't worked with Sphinx before, you should first read their `quickstart <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ guide.
+TARDIS uses the popular Python documentation generator `Sphinx <https://www.sphinx-doc.org/>`_. Sphinx translates a set of source files (often written in reStructuredText or Jupyter notebooks, see below) to HTML files, automatically producing cross-references, indices, etc. If you haven't worked with Sphinx before, you should first read their `quickstart <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ guide.
 
 
 Documenting the code you write
 ==============================
 
-When making or adding changes to the functionality of an aspect of TARDIS, an example notebook or .rst file should be created to demonstrate how it works. A good example of this is the `quickstart notebook <https://tardis-sn.github.io/tardis/quickstart/quickstart.html>`_ which gives an overview of how TARDIS works. You will also need to add the .rst (or notebook) file's path within the relevant ``index.rst`` file - it is usually present within the same (or parent) directory where you've created your file. For example, when the documentation was created for `research papers that used TARDIS <https://tardis-sn.github.io/tardis/research/research_done_using_TARDIS/research_papers.html>`_, the toctree in `this index file <https://github.com/tardis-sn/tardis/blob/master/docs/research/index.rst>`_ was edited to add the path to the new .rst file.
+When making or adding changes to the functionality of an aspect of TARDIS, an .rst file or notebook should be created to demonstrate how it works, and that page must then be included in the documentation. This is described in detail in the following sections.
```

Comment:
```suggestion
When making or adding changes to the functionality of an aspect of TARDIS, an .rst file or notebook (.ipynb file) should be created to demonstrate how it works, and that page must then be included in the documentation. This is described in detail in the following sections.
```

Reply:
Should we write extensions with backticks?